{{cookiecutter.project_slug}}/tests/test_{{cookiecutter.project_slug}}.py

```diff
@@ -26,30 +26,26 @@
 
 
 {% if cookiecutter.use_pytest == 'y' -%}
-class Test{{ cookiecutter.project_slug|title }}(object):
+@pytest.fixture
+def some_fixture():
+    pass
```

Comment:
I agree, however instead of using a database transaction what do you think about creating a fixture that uses the requests library and returns a requests object? Or maybe we could include both the database transaction and the requests object to show that more than one fixture can be used?